src/components/RestaurantReview.js

```diff
@@ -0,0 +1,125 @@
+import { useState } from "react";
+import { Star } from "react-star";
+import FileBase64 from "react-file-base64";
+
+const RestaurantReview = ({
+  restaurantName,
+  restaurantPhone,
+  restaurantAddress,
+  restaurantWebsite,
+}) => {
+  const [textReview, setTextReview] = useState("");
+  const [deliciousnessScore, setDeliciousnessScore] = useState(0);
+  const [serviceScore, setServiceScore] = useState(0);
+  const [experienceScore, setExperienceScore] = useState(0);
+  const [pricingScore, setPricingScore] = useState(0);
+  const [pricingValue, setPricingValue] = useState(0);
+  const [base64Img, setBase64Img] = useState([]);
+
+
+  const onSubmit = (e) => {
+    e.preventDefault();
+
+    const restaurantReview = {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Accept: "application/json",
+      },
+      body: JSON.stringify({
+        url: "/restaurant/review/save",
+        restaurantName: restaurantName,
+        restaurantPhone: restaurantPhone,
+        restaurantAddress: restaurantAddress,
+        restaurantWebsite: restaurantWebsite,
+        review: textReview,
+        deliciousnessScore: deliciousnessScore,
+        serviceScore: serviceScore,
+        experienceScore: experienceScore,
+        pricingScore: pricingScore,
+        pricingValue: pricingValue,
+        images: base64Img.length > 0 ? base64Img[1] : [] //TODO: add validation for empty array
+      }),
+    };
+
+    saveReview(restaurantReview);
+  };
+
+  const saveReview = async (restaurantReview) => {
+    // console.log(restaurantReview.body);
+    // console.warn(base64Img[1][0].base64);
+    const res = await fetch(
+      "http://localhost:80/api/index.php",
+      restaurantReview
+    );
+
+
+    const data = await res.json();
+    console.log(data['response']);
+    if (Number.isInteger(parseInt(data['response']))) {
+      alert("Review Saved");
+    }
```

Comment:
Instead of alert, maybe we can display a message on the HTML  DOM for better User Experience